docs-ref-conceptual/get-started-tutorial-1-prepare-environment.md

```diff
@@ -0,0 +1,183 @@
+
+When working with variables, Bash and PowerShell syntax is different. Copy the correct script for your environment.
+
+# [Bash](#tab/bash)
+
+```azurecli-interactive
+let "randomIdentifier=$RANDOM*$RANDOM"
+location="eastus"
+resourceGroup="msdocs-tutorial-rg-$randomIdentifier"
+az group create --name $resourceGroup --location $location --output json
+```
+
+# [PowerShell](#tab/powershell)
+
+```azurecli-interactive
+$randomIdentifier = (New-Guid).ToString().Substring(0,8)
+$location="eastus"
+$resourceGroup="msdocs-tutorial-rg-$randomIdentifier"
+az group create --name $resourceGroup --location $location --output json
+```
+
+---
+
+Bash and PowerShell console output:
+
+```output
+{
+  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/msdocs-tutorial-rg-000000000",
+  "location": "eastus",
+  "managedBy": null,
+  "name": "msdocs-tutorial-rg-000000000",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+## Set environment variables
+
+The Azure CLI offers several options to allow you to reuse common parameter values. These default values are stored in environment variables and are unique for each login.
+
+1. Set your default resource group.
+
+   ```azurecli-interactive
+   az config set defaults.group=<msdocs-tutorial-rg-0000000>
+   ```
+
+1. Set multiple environment variables at once.
+
+   When working with the Azure CLI, many parameters take multiple values separated by a space. Configuration values is one such instance. This example sets both the `.location` and `.group` defaults that will be used by the `--location` and `--resource-group` parameters of every Azure CLI command.
+
+   ```azurecli-interactive
+   az config set defaults.location=westus2 defaults.group=<msdocs-tutorial-rg-0000000>
+   ```
+
+1. Set your default output.
+
+   When you chose to work in Azure Cloud Shell, or installed the Azure CLI locally, the default output is automatically set to `json`.  However, this is one of the most important defaults to understand and set.  **Output determines what appears on your console and what is written to your log file.** Always use an output of `none` when you are creating resources that return keys, passwords and secrets.
+
+   ```azurecli-interactive
+   az config set core.output=none
+   ```
+
+   In this tutorial we are not working with secrets.  Set the default back to `json` so you can see the returned output of each reference command in this tutorial.
+
+   ```azurecli-interactive
+   az config set core.output=json
+   ```
+
+1. Learn to use `az init`.
+
+   The Azure CLI has a reference command that walks you through configuring your environment. Type `az init` in your console and press <kbd>Enter</kbd>.  Follow the prompts provided. (The first nice thing about [az init](/cli/azure/reference-index#az-init) is that it gives you all of your current settings!)
```

Comment:
Consider adding the portion in parentheses in a separate note, it might be helpful to highlight cool benefits the user should know

Reply:
good suggestion.  I put in an ```ouput code block.